<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newprojectdialog</class>
 <widget class="QDialog" name="newprojectdialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>541</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="what_type_of_data_Label">
     <property name="minimumSize">
      <size>
       <width>281</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>WHAT TYPE OF DATA DO YOU HAVE?</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>480</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="2">
       <widget class="QToolButton" name="onearm_single_reg_coef_Button">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>65</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>65</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
        <property name="text">
         <string>single regression
coefficient</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QToolButton" name="twoarm_smds_Button">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>65</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>65</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
        <property name="text">
         <string>Standardized
mean differences</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QToolButton" name="onearm_generic_effect_size_Button">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>65</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>65</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
        <property name="text">
         <string>generic
effect size</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="arrowType">
         <enum>Qt::NoArrow</enum>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QToolButton" name="onearm_mean_Button">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>65</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>65</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
        <property name="text">
         <string>mean</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QToolButton" name="onearm_proportion_Button">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>65</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>65</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
        <property name="text">
         <string>proportion</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Data on two or more groups per study</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QToolButton" name="twoarm_proportions_Button">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>65</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>65</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
        <property name="text">
         <string>proportions</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Data on test performance</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QToolButton" name="diagnostic_Button">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>65</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>65</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
        <property name="text">
         <string>diagnostic</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="6" column="0" colspan="6">
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="6">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QToolButton" name="twoarm_means_Button">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>65</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>65</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
        <property name="text">
         <string>means</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="0" colspan="5">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>One piece of data from each study or studies with one group</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>onearm_proportion_Button</zorder>
     <zorder>line_2</zorder>
     <zorder>twoarm_proportions_Button</zorder>
     <zorder>diagnostic_Button</zorder>
     <zorder>label_2</zorder>
     <zorder>label_4</zorder>
     <zorder>label_5</zorder>
     <zorder>line_4</zorder>
     <zorder>onearm_mean_Button</zorder>
     <zorder>onearm_single_reg_coef_Button</zorder>
     <zorder>onearm_generic_effect_size_Button</zorder>
     <zorder>twoarm_means_Button</zorder>
     <zorder>twoarm_smds_Button</zorder>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="outcome_name_Frame">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>What is the name of your outcome?</string>
        </property>
        <property name="buddy">
         <cstring>outcome_name_LineEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="outcome_name_LineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="helpLabel">
     <property name="text">
      <string>&lt;a href=&quot;http://tuftscaes.org/open_meta/help/openMA_help.html&quot;&gt;help&lt;/a&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>newprojectdialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>451</x>
     <y>482</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>535</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>newprojectdialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>489</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>531</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
